Contributing.md

```diff
@@ -1,21 +1,20 @@
 # Contributing and Forking This Tool
 
-You can use this tool as currently designed for your own event. At present, it pushes all seed nominations to a Google spreasheet owned by the EDGI group in Toronto. If you would like to push to your own spreasheet, the process is pretty simple, but requires small modifications to the sourcecode. 
+You can use this tool as currently designed for your own event. At present, it pushes all seed nominations to a Google spreasheet owned by the EDGI group in Toronto. If you would like to push to your own spreasheet, the process is pretty simple, but requires small modifications to the sourcecode.
 
 1. Create your own Google Form.  You can copy [ours](https://docs.google.com/forms/d/1kuwxu2lXYSRpkwBj4o9kwjURZL3hgk-mSFoK4qkC4ZI/edit), or create your own from scratch
     * Once you've created a form, Google will automagically create a spreadsheet in which to store the responses. You can also use the fancy response visualizations tool to get some basic information about submissions.
-  * Make note of the form URL. In `popup.js`, update the `GOOGLE_FORMS_URL` variable with the new URL, taking care to **replace the final `edit` with `/formResponse?ifq`**. 
-  
+  * Make note of the form URL. In `popup.js`, update the `GOOGLE_FORMS_URL` variable with the new URL, taking care to **replace the final `edit` with `/formResponse`**.
+
   The line you're looking for looks like this:
   ```js
-  var GOOGLE_FORMS_URL = 'https://docs.google.com/forms/d/1kuwxu2lXYSRpkwBj4o9kwjURZL3hgk-mSFoK4qkC4ZI/formResponse?ifq';
+  const GOOGLE_FORMS_URL = 'https://docs.google.com/forms/d/1kuwxu2lXYSRpkwBj4o9kwjURZL3hgk-mSFoK4qkC4ZI/formResponse';
```

Comment:
We don't use `const` https://github.com/edgi-govdata-archiving/presidential-harvest-nomination-tool/blob/master/popup.js#L24